annotations/models.py

```diff
@@ -85,6 +85,36 @@ def __str__(self):
         return '{} - {}'.format(self.corpus.title, self.title)
 
 
+class LabelKey(models.Model):
+    """Used to define what kind of labels should be used per corpus,
+    and to group labels from different languages"""
+    title = models.CharField(max_length=200, unique=True)
+
+    corpora = models.ManyToManyField(Corpus, related_name='label_keys')
+
+    class Meta:
+        verbose_name_plural = 'Label Keys'
+
+    def symbol(self):
```

Comment:
Why not `lower` instead of `symbol`?

Reply:
could be changed, but I thought `symbol` better conveys what this is used for.